src/main/java/org/folio/innreach/domain/entity/TransactionHold.java

```diff
@@ -83,4 +83,7 @@
   @Column(name = "folio_item_barcode")
   protected String folioItemBarcode;
 
+  @Column(name = "folio_item_call_number")
+  protected String folioItemCallNumber;
```

Comment:
I'm not sure why would we need a field for `folioItemCallNumber`. Is there a requirement for it?

Reply:
@oliinyko, Update transaction callNumber and the FOLIO item call number to the payload callNumber (if present)
Since we don't have this field, we need to add it, I guess

Reply:
I assume this means that we need to update the inventory item's callNumber (which we already do) 

>... and the FOLIO item call number to the payload callNumber (if present)

https://github.com/folio-org/mod-inn-reach/blob/3fba3f92a2e5d17e5b5e5dd4451f8b4263d37d30/src/main/java/org/folio/innreach/domain/service/impl/CirculationServiceImpl.java#L313-L315

Reply:
@oliinyko It looks like we don't need to do anything here. Since it already uses the PUT method, everything else has already been implemented.

Reply:
seems like that)